libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/SpecTranslate/Conway.hs

```diff
@@ -1292,16 +1286,40 @@ instance
   where
   type SpecRep (EpochState era) = Agda.EpochState
 
-  toSpecRep (EpochState {esLState = esLState@LedgerState {..}, ..}) =
+  toSpecRep (EpochState {esLState = esLState@LedgerState {lsUTxOState}, ..}) =
     Agda.MkEpochState
       <$> toSpecRep esAccountState
+      <*> toSpecRep esSnapshots
       <*> toSpecRep esLState
       <*> toSpecRep enactState
       <*> toSpecRep ratifyState
     where
       enactState = mkEnactState $ utxosGovState lsUTxOState
       ratifyState = RatifyState enactState mempty mempty False
 
+instance SpecTranslate ctx (SnapShots c) where
```

Comment:
Just for my information - is this mandated by the update of the executable spec SRP? Or is it something additional, not directly related to the update?

Reply:
This is due to the change in the SRP. It now requires the snapshots to be here.